.github/workflows/wasm32.yml

```diff
@@ -0,0 +1,43 @@
+name: WebAssembly CI
+
+on: [push, pull_request]
+
+env:
+  SPEC_SPLIT_DOTS: 160
+
+jobs:
+  wasm32-test:
+    runs-on: ubuntu-latest
+    container: crystallang/crystal:1.6.1-build
+    steps:
+      - name: Download Crystal source
+        uses: actions/checkout@v2
+
+      - name: Install wasmtime
+        run: |
+          apt-get update
+          apt-get install -y curl
+          touch ~/.bashrc
+          curl https://wasmtime.dev/install.sh -sSf | bash -s - --version v2.0.0
```

Comment:
Would it make sense to use an install action?
I could only find https://github.com/marketplace/actions/set-up-wasmtime though, which doesn't seem to be maintained (at least there's been no update for 2 years).

Reply:
I wasn't updated recently but seems to work well. Added!